script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -66,23 +66,23 @@ I whip my sluggish body back into shape and close the curtains.
 [sha:ef8309e8256a36bf0262533bb7b46760060b6e15]{
 -- Page 7, Offset 8604.
 -- やれる事をやって、ようやく緊張も解けたのだろう。
-After doing what I could, I should finally be able to relax.
+I've done everything I could. Now, I should finally be able to relax.
 }
 [sha:7cf739aa2eebbbfc57ec4999f7a27d86a0d01cbc]{
 -- Page 7, Offset 8605.
 -- 座ろうとした途中、膝から力が抜けて、どすん、と床に倒れこんだ。
-But on my way to go sit down, my knees give out and I fall to the floor with a thud.
+Just as I'm about to sit down, my knees give out and I fall to the floor with a thud.
```

Comment:
Think the original makes a bit more sense here, he's still over by the curtains and is on his way to a place to sit down. If here were already about to sit down and collapsed, he'd probably already be over a chair